DiscordBot/Commands/EventCommands.cs

```diff
@@ -70,50 +69,69 @@ public async Task Event(CommandContext context)
             switch (eventOperation)
             {
                 case "create":
-                    await CreateEvent(context, interactivity);
+                    await CreateEventName(context);
                     break;
                 case "list":
                     await ListEvents(context);
                     break;
                 case "remove":
-                    await RemoveEvent(context, interactivity);
+                    await RemoveEvent(context);
                     break;
                 case "show":
-                    await ShowEvent(context, interactivity);
+                    await ShowEvent(context);
                     break;
                 case "edit":
-                    await EditEvent(context, interactivity);
+                    await EditEvent(context);
                     break;
                 case "start":
-                    await CreateSignupSheet(context, interactivity);
+                    await CreateSignupSheet(context);
                     break;
             }
         }
 
-        public async Task CreateEvent(CommandContext context, InteractivityExtension interactivity)
+        [Command("create")]
+        public async Task CreateEventName(CommandContext context)
```

Comment:
Why the rename? It's not just creating the event name, it's creating the whole event...

Reply:
This is very true, should I change the names of all of them back to just "CreateEvent" as they all end up doing the same thing (using overloads)

Reply:
Yes, I think that sounds sensible 🙂